.dockerignore

```diff
@@ -0,0 +1,13 @@
+*/log
```

Comment:
Should this be the whole log directory? It suprised me that the tmp directory is ignored but not log

Reply:
Oh this means any file or directory called log that's exactly one level below the directory that contains `.dockerignore`. Some of the other ones probably need updating.